recipe/meta.yaml

```
@@ -2,63 +2,103 @@
 {% set version = "0.1.0a14.post1" %}
 
 package:
-  name: {{ name|lower }}
+  name: nurikit-split
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/nurikit-{{ version }}.tar.gz
   sha256: af4d7702fa51671d3bbe5e17474d1116e6763b5a466b49f3be5861b83d422411
 
 build:
-  script: |
-    export CMAKE_ARGS="${CMAKE_ARGS} -GNinja -DNURI_PREBUILT_ABSL=OFF -DBUILD_TESTING=OFF"
-    {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
   skip: true  # [win]
-  skip: true  # [linux and not x86_64]
-
-  ignore_run_exports_from:
-    # numpy used for typing, binary works with any version of numpy (pybind11)
-    - numpy
-    # We use headers only, but need -devel instead -headers for cmake
-    - libboost-devel
-  ignore_run_exports:
-    - numpy *
-
-requirements:
-  build:
-    - {{ compiler('c') }}
-    - {{ stdlib('c') }}
-    - {{ compiler('cxx') }}
-    - cmake >=3.16
-    - ninja
-    - python                              # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
-    - pybind11                            # [build_platform != target_platform]
-    - pybind11-stubgen                    # [build_platform != target_platform]
-    - numpy *                             # [build_platform != target_platform]
-  host:
-    - python
-    - libboost-devel >=1.82
-    - libabseil >=20240116
-    - eigen >=3.4
-    - spectralib >=1.0
-    - pybind11 >=2.12  # For numpy 2
-    - scikit-build-core >=0.5
-    - pybind11-stubgen
-    - numpy *
-    - pip
-  run:
-    - python
-    - numpy <3.0
-
-test:
-  imports:
-    - nuri
-  commands:
-    - pip check
-  requires:
-    - pip
+
+outputs:
+  - name: libnurikit
+
+    build:
+      script: |
+        cmake -S. -B build ${CMAKE_ARGS} -GNinja \
+          -DNURI_BUILD_PYTHON=OFF \
+          -DNURI_PREBUILT_ABSL=OFF \
+          -DNURI_FORCE_VERSION="${PKG_VERSION}" \
+          -DBUILD_TESTING=OFF
+        cmake --build build --target all -j "${CPU_COUNT}"
+        cmake --build build --target install/strip
+
+      run_exports:
+        - ${{ pin_subpackage("libnurikit", exact=True) }}
+
+      ignore_run_exports_from:
+        # We use headers only, but need -devel instead -headers for cmake
+        - libboost-devel
+
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+        - {{ compiler('cxx') }}
+        - cmake >=3.16
+        - ninja
+      host:
+        - libboost-devel >=1.82
+        - libabseil >=20240116
+        - eigen >=3.4
+        - spectralib >=1.0
+
+    test:
+      commands:
+        - "$PREFIX/bin/nuri"
```

```suggestion
        - nuri --help
```

is there a reason why this doesn't work? the general idea is that you want to test that the command is found by the shell the user uses. The user doesn't want to to `${CONDA_PREFIX}/bin/nuri` instead, they just want to type nuri